Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenWindowControllerIOS.mm

```diff
@@ -61,6 +61,11 @@
 
 #if PLATFORM(VISION)
 #import "MRUIKitSPI.h"
+#import "WKSPreviewWindowController.h"
+#import <pal/ios/QuickLookSoftLink.h>
+#import "WebKitSwiftSoftLink.h"
```

Comment:
When we import a soft-linking header we need to mark the file with `@no-unify` in SourcesCocoa.txt and add it to the WebKit target in Xcode.

Reply:
Should the header also be added to the WebKit target? (the impl already was)